Composer/packages/extensions/visual-designer/src/components/renderers/ElementWrapper.tsx

```diff
@@ -26,10 +27,34 @@ const nodeBorderDoubleSelectedStyle = css`
 export interface ElementWrapperProps {
   id: string;
   tab?: string;
+  titleInHeader?: boolean;
   onEvent: (eventName: NodeEventTypes, eventData: any) => any;
 }
 
-export const ElementWrapper: FC<ElementWrapperProps> = ({ id, tab, onEvent, children }): JSX.Element => {
+function checkHasProps(node: ReactNode): node is ReactElement {
+  return (node as ReactElement).props != null;
+}
+
+function extractNodeTitle(node: ReactNode, titleInHeader: boolean): string {
+  if (node == null || typeof node !== 'object') {
```

Comment:
`===` for `node==null`? Or does it not matter for `null`?

Reply:
Checks for `null` and `undefined` still just use `==` by convention.